Sources/CodegenCLI/Commands/Initialize.swift

```diff
@@ -106,36 +106,52 @@ public struct Initialize: ParsableCommand {
 
 // MARK: - Private extensions
 
-fileprivate extension ApolloCodegenConfiguration {
-  static var `default`: ApolloCodegenConfiguration {
+extension ApolloCodegenConfiguration {
+  static var minimalJSON: String {
     #if COCOAPODS
-    ApolloCodegenConfiguration(
-      schemaName: "GraphQLSchemaName",
-      input: .init(
-        schemaPath: "schema.graphqls"
-      ),
-      output: .init(
-        schemaTypes: .init(path: "./", moduleType: .swiftPackageManager)
-      ),
-      options: .init(cocoapodsCompatibleImportStatements: true)
-    )
+      minimalJSON(supportCocoaPods: true)
     #else
-    ApolloCodegenConfiguration(
-      schemaName: "GraphQLSchemaName",
-      input: .init(
-        schemaPath: "schema.graphqls"
-      ),
-      output: .init(
-        schemaTypes: .init(path: "./", moduleType: .swiftPackageManager)
-      )
-    )
+      minimalJSON(supportCocoaPods: false)
     #endif
   }
 
-  func encoded() throws -> Data {
-    let encoder = JSONEncoder()
-    encoder.outputFormatting = [.prettyPrinted]
-
-    return try encoder.encode(self)
+  static func minimalJSON(supportCocoaPods: Bool) -> String {
+    let cocoaPodsOption = supportCocoaPods ? """
```

Comment:
lol I love this hack. If `TemplateString` was a separate library, we could use that here ;) but this works fine!